config/testreport/css/default.css

```diff
@@ -1727,6 +1729,36 @@ tr:hover td.a100
     background-color: var(--web-vital-color-poor);
 }
 
+/*
+ * ----------------------------------------------------
+ * Load Distribution
+ * ----------------------------------------------------
+ */
+
+table tbody tr:nth-child(odd) .load-meter {
+	background: -webkit-linear-gradient(left, var(--load-bar-color) calc(var(--loadp) * 1%), #f9f9fc calc(var(--loadp) * 1%));
+    background:    -moz-linear-gradient(left, var(--load-bar-color) calc(var(--loadp) * 1%), #f9f9fc calc(var(--loadp) * 1%));
+    background:     -ms-linear-gradient(left, var(--load-bar-color) calc(var(--loadp) * 1%), #f9f9fc calc(var(--loadp) * 1%));
+    background:      -o-linear-gradient(left, var(--load-bar-color) calc(var(--loadp) * 1%), #f9f9fc calc(var(--loadp) * 1%));
```

Comment:
Do we really need the non-standard gradient definitions? Gradient support is now widely available. Also, we use gradients elsewhere in this file without the non-standard fallbacks.